jbpm-workitems/jbpm-workitems-rest/pom.xml

```diff
@@ -19,6 +19,16 @@
   <properties>
     <java.module.name>org.jbpm.workitem.rest</java.module.name>
   </properties>
+  
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.mock-server</groupId>
```

Comment:
dependencies needs to be managed.

Reply:
Depends on [jbpm#1355](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1355)